main_app/models.py

```diff
@@ -17,11 +18,12 @@ def get_absolute_url(self):
     # Create your models here.
 class Item(models.Model):
     name = models.CharField(max_length=100)
-    quantity_current = models.IntegerField()
+    quantity_current = models.IntegerField(default=0)
     quantity_min = models.IntegerField(default = 1)
     quantity_max = models.IntegerField()
-    description = models.TextField(max_length=250)
+    description = models.TextField(max_length=1000)
```

Comment:
Was copy/pasting a description from Amazon. Realised that 250 characters isn't really enough for a decent description of the item.